### ISSUE TITLE
Progressive Depth and Distribution or Ores Mod Changes

### ISSUE THREAD

**Issue by Silverbane** [Silverbane/SilverSkyEnt-comSpaceEngineers#24]
Double depth (thickness) on all ores.
Figure out why Ores are spawning everywhere on the surface on earth.
Maybe Replace all Ore maps with my own versions.

**Comment by billedwardz** [comment]
I'm only for this if you can halve the frequency of the ore spawns. I really like that it takes effort to get resources. 

If you're worried about new players not returning to the server because of how hard it is to find ore, then I would just suggest a slight buff to the vanilla ore detector.

**Comment by Silverbane** [comment]
Yeah thinking of doing it with the possibility of editing the ore mapping
files to reduce how many ores spawn
